core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -594,6 +594,7 @@ private[kafka] object Processor {
   val IdlePercentMetricName = "IdlePercent"
   val NetworkProcessorMetricTag = "networkProcessor"
   val ListenerMetricTag = "listener"
+  val SocketServerMetricsGroup = "socket-server-metrics"
```

Comment:
This should probably be in the SocketServer object, right?

Reply:
That makes better sense, thanks